dali/ft/filecopy.cpp

```diff
@@ -830,7 +840,12 @@ void FileSprayer::beforeTransfer()
                     splitUNCFilename(remoteFilename.str(), &remoteDirectory, &remoteDirectory, NULL, NULL);
 
                     Owned<IFile> dir = createIFile(remoteDirectory.str());
+                    bool modPerms = false;
+                    if (!dir->exists() && (umask != -1))
+                        modPerms = true;
                     dir->createDirectory();
+                    if (modPerms)
+                        dir->setFilePermissions(~umask&0777);
```

Comment:
I am probably confusing how file/dir 'mode' is treated against process mask, vs chmod to that can give less restrictive perms to file...

If so, ignore prev. comment.